sonic-net/sonic-buildimage
# reset-factory more robust with exec reboot

#### Why I did it
reset-factory dose not flush config_db.json when executed from a console terminal.

##### Work item tracking
- Microsoft ADO **(number only)**: None

#### How I did it
Replace "reboot" with "exec reboot" in the reset-factory script.

#### How to verify it
Create a VLAN and save the config.
Execute reset-factory from a console terminal.
After reboot check the VLAN should not exist.



#### Which release branch to backport (provide reason below if selected)



#### Tested branch (Please provide the tested image version)




#### Description for the changelog

reset-factory more robust with exec reboot



#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
